android/src/main/java/com/snowplowanalytics/react/tracker/RNSnowplowTrackerModule.java

```diff
@@ -42,6 +44,8 @@ public void initialize(String endpoint, String method, String protocol,
         this.tracker = Tracker.init(new Tracker
                 .TrackerBuilder(this.emitter, namespace, appId, this.reactContext)
                 .base64(false)
+                .mobileContext(true)
+                .screenviewEvents(options.hasKey("autoScreenView") ? options.getBoolean("autoScreenView") : false)
```

Comment:
AutoScreenView is a very handy feature. Thanks a lot!

Reply:
👍 Unfortunately I'm not terribly sure how it's going to behave in a RN app - normally it depends on mainActivity/view Controller, but I think RN may handle transitions differently.

We might look at some point in the future to instrument a React-Native based autotracking, but the MVP for v1 of the tracker will probably require manual instrumentation unless you're using those native transitions.

Reply:
OK, then. Manual tracking will help until this is sorted out.